.github/workflows/kube-linter.yaml

```diff
@@ -25,7 +25,7 @@ jobs:
       - name: Run kustomize build
         run: |
           find argo-cd-apps components -name 'kustomization.yaml' ! -path '*/k-components/*' | \
-            xargs -I {} -n1 -P0  bash -c 'dir=$(dirname "{}"); output_file=$(echo $dir | tr / -)-kustomization.yaml; if ! log=$(kustomize build "$dir" -o "kustomizedfiles/$output_file" 2>&1); then echo "Error when running kustomize build for $dir: $log" && exit 1;fi'
+            xargs -I {} -n1 -P8  bash -c 'dir=$(dirname "{}"); output_file=$(echo $dir | tr / -)-kustomization.yaml; if ! log=$(kustomize build "$dir" -o "kustomizedfiles/$output_file" 2>&1); then echo "Error when running kustomize build for $dir: $log" && exit 1;fi'
```

Comment:
your description says reduce from 8 to 4, but your code change is from -P0 to -P8

things are clean so I'm fine merging, but I'm at least confused where the numbers come from

/lgtm

Reply:
Thanks for pointing this out. In fact, it is 8, I did so many debugging changes and I obviously opened the PR too quickly :). let me fix that

Reply:
too late... it got merged

The change is ok, only the commit message is not but it is too late now